tools/packages.config

```diff
@@ -2,12 +2,12 @@
 <packages>
     <package id="Cake" version="0.17.0" />
     <package id="Microsoft.Build.Runtime" version="15.1.548" />
-    <package id="Microsoft.NET.Sdk" version="1.0.0-alpha-20170217-2" />
+    <package id="Microsoft.NET.Sdk" version="1.0.0-alpha-20170224-6" />
     <package id="Microsoft.NET.Sdk.Web" version="1.0.0-alpha-20170130-3-281" />
     <package id="Microsoft.NET.Sdk.Publish" version="1.0.0-alpha-20170130-3-281" />
     <package id="Microsoft.NET.Sdk.Web.ProjectSystem" version="1.0.0-alpha-20170130-3-281" />
-    <package id="Microsoft.Net.Compilers" version="2.0.0-rc4-61325-08" />
-    <package id="NuGet.Build.Tasks" version="4.0.0-rtm-2283" />
-    <package id="NuGet.Build.Tasks.Pack" version="4.0.0-rtm-2283" />
+    <package id="Microsoft.Net.Compilers" version="2.0.1" />
```

Comment:
I asked the same question in my team room. Somebody failed about five months ago I guess. :smile: